akka-actor-typed/src/main/scala/akka/actor/typed/scaladsl/AskPattern.scala

```diff
@@ -7,20 +7,41 @@ package akka.actor.typed.scaladsl
 import java.util.concurrent.TimeoutException
 
 import scala.concurrent.Future
-
 import com.github.ghik.silencer.silent
-
 import akka.actor.{ Address, RootActorPath }
 import akka.actor.typed.ActorRef
 import akka.actor.typed.ActorSystem
 import akka.actor.typed.RecipientRef
 import akka.actor.typed.Scheduler
+import akka.actor.typed.StatusResponse
+import akka.actor.typed.StatusResponse
+import akka.actor.typed.StatusResponse
+import akka.actor.typed.StatusResponse
+import akka.actor.typed.StatusResponse
+import akka.actor.typed.StatusResponse
+import akka.actor.typed.StatusResponse
+import akka.actor.typed.StatusResponse
+import akka.actor.typed.StatusResponse
+import akka.actor.typed.StatusResponse
+import akka.actor.typed.StatusResponse
+import akka.actor.typed.StatusResponse
+import akka.actor.typed.StatusResponse
+import akka.actor.typed.StatusResponse
+import akka.actor.typed.StatusResponse
+import akka.actor.typed.StatusResponse
+import akka.actor.typed.StatusResponse
+import akka.actor.typed.StatusResponse
+import akka.actor.typed.StatusResponse
```

Comment:
Ok, I understood you want to import it.

Reply:
I really wanted to make sure it is imported.

Reply:
Thanks IntelliJ